app/models/manageiq/providers/azure/cloud_manager/provision/cloning.rb

```diff
@@ -183,5 +183,9 @@ def start_clone(clone_options)
 
       File.join(azure.subscription_id, vm.resource_group.downcase, vm.type.downcase, vm.name)
     end
+  rescue StandardError => e
+    return if phase_context[:clone_options].nil?
+    signal :clone_failure_cleanup
+    raise
```

Comment:
`signal` is essentially a method call so this raise won't necessarily be reachable